packages/admin/admin/src/mui/menu/ItemGroup.tsx

```diff
@@ -36,15 +40,31 @@ const styles = (theme: Theme) =>
         titleContainerMenuOpen: {
             justifyContent: "flex-start",
             padding: theme.spacing(2, 4),
+            alignItems: "center",
+        },
+        tooltipHelperBox: {
+            display: "flex",
+            alignItems: "center",
+            cursor: "pointer",
         },
     });
 
 export interface MenuItemGroupProps {
     title: React.ReactNode;
     shortTitle?: React.ReactNode;
+    helperIcon?: React.ReactElement<React.SVGProps<SVGSVGElement>>;
+    helperText?: React.ReactNode;
```

Comment:
Instead of having two props, why not add one `helper?: ReactNode` and let developers decide what they want to render?

Reply:
Because we can't expect a dev to know this workaround 🤷🏻‍♂️ 
<img width="514" alt="image" src="https://github.com/vivid-planet/comet/assets/56400587/dc81baa5-c681-41a7-88dd-e8ca6edf2a23">

Reply:
<img width="912" alt="image" src="https://github.com/vivid-planet/comet/assets/48853629/7aca3b70-7cab-41ce-9d97-940dc1f999f7">

I just tried this in the MUI docs, and it works perfectly fine with the icon?

Reply:
With MUI icons but not with our custom ones...

Reply:
Then we should fix that 😉 

https://github.com/vivid-planet/comet/pull/1663